### DISCUSSION TITLE
Change Owner/Transfer Ownership

### DISCUSSION THREAD

**Discussion by Hutch79** [GriefPrevention/GriefPrevention#1651] [General]
Hay
I have no clue if this question is asked in the right spot. If not, please tell me.

-------

Is it somehow possible for players to change the owner?

I already searched in the command list on GitHub and Bukkit, but I unfortunately found nothing.

Thanks already for your help.

**Comment by Jikoo** [comment]
Regular players cannot. Admins can using `/transferclaim`

**Reply by Hutch79** [reply]
Oh my goodness ._.

Seriously? I searched the hole time for something like chanceowner XD

Thanks very much!

**Reply by Hutch79** [reply]
Cloud you maybe help me with another problem as well?

I want to turn off that users can build in the wild.
So that they can only build in claimed regions.

But even after reading the config file twice, I haven't found anything.
Maybe I'm just blind or something, but after two days I'll give up

**Reply by Jikoo** [reply]
Sounds like you want claims mode `SurvivalRequiringClaims`: https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on

**Reply by Hutch79** [reply]
That was exactly what I searched for!

Thank you very much!
